packages/shared/routes/onboarding/views/ledger-setup/views/AccountIndexView.svelte

```diff
@@ -9,8 +9,6 @@
     import { LedgerAppName } from '@lib/typings/ledger'
 
     const dispatch = createEventDispatcher()
-    const min = 0
-    const max = 2147483647
 
     let busy = false
```

Comment:
Min / max removed but still expected in the file?

Reply:
Potentially they were unused, but I'll double check that that file is okay.